src/cr/cube/matrix/cubemeasure.py

```diff
@@ -968,7 +1107,7 @@ def _regular_matrix_class(dimension_types):
         )
 
     @staticmethod
-    def _regular_matrix_counts_slice(cube, slice_idx):
+    def _regular_matrix_slice(cube, slice_idx):
```

Comment:
The `regular_` prefix no longer makes any sense here because there is no "non-regular" counterpart. Also, this is not a "matrix-slice", it is the "slice-expression" that digs out the right portion of the overall cube values. So something like "._cube_slice_expression" is more appropriate.